api/envoy/extensions/network/dns_resolver/getaddrinfo/v3/getaddrinfo_dns_resolver.proto

```diff
@@ -0,0 +1,30 @@
+syntax = "proto3";
+
+package envoy.extensions.network.dns_resolver.getaddrinfo.v3;
+
+import "udpa/annotations/status.proto";
+
+option java_package = "io.envoyproxy.envoy.extensions.network.dns_resolver.getaddrinfo.v3";
+option java_outer_classname = "GetaddrinfoDnsResolverProto";
+option java_multiple_files = true;
+option go_package = "github.com/envoyproxy/go-control-plane/envoy/extensions/network/dns_resolver/getaddrinfo/v3;getaddrinfov3";
+option (udpa.annotations.file_status).package_version_status = ACTIVE;
+
+// [#protodoc-title: getaddrinfo DNS resolver]
+// [#extension: envoy.network.dns_resolver.getaddrinfo]
+
+// Configuration for getaddrinfo DNS resolver. This resolver will use the system's getaddrinfo()
+// function to resolve hosts.
+//
+// .. attention::
+//
+//   This resolver uses a single background thread to do resolutions. As such, it is not currently
+//   advised for use in situations requiring a high resolution late. A thread pool can be added
```

Comment:
in situations requiring a high resolution late -> in situations requiring a high resolution rate?